README.md

```diff
@@ -1,23 +1,21 @@
-# Helper libraries for working with typed data lists.
+[![Dart CI](https://github.com/dart-lang/typed_data/actions/workflows/test-package.yml/badge.svg)](https://github.com/dart-lang/typed_data/actions/workflows/test-package.yml)
+[![pub package](https://img.shields.io/pub/v/typed_data.svg)](https://pub.dev/packages/typed_data)
+[![package publisher](https://img.shields.io/pub/publisher/typed_data.svg)](https://pub.dev/packages/typed_data/publisher)
+
+Helper libraries for working with typed data lists.
 
 The `typed_data` package contains utility functions and classes that makes working with typed data lists easier.
 
 ## Using
 
-The `typed_data` package can be imported as
+The `typed_data` package can be imported using:
 
 ```dart
 import 'package:typed_data/typed_data.dart';
 ```
 
-## Typed buffers: Growable typed data lists
+## Typed buffers
 
-Typed buffers are contains growable lists backed by typed arrays.
+Typed buffers are growable lists backed by typed arrays.
 These are similar to the growable lists returned by `List()`, 
```

Comment:
While you are here, `List()` no longer works, so maybe;

> ... the growable lists created by `<int>[]` or `<double>[]`, but

Reply:
👍 , done